blueledger-webapp/lib/api/withAuth.ts

```
@@ -0,0 +1,19 @@
+import { auth } from '@/lib/auth/auth';
+import { NextAuthRequest } from 'next-auth';
+import { NextResponse } from 'next/server';
+
+type Handler<P = undefined> = (
+  req: NextAuthRequest,
+  // eslint-disable-next-line
+  context: P extends undefined ? {} : { params: P }
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)